samples/peripheral/802154_phy_test/src/periph_proc.c

```diff
@@ -182,7 +192,7 @@ bool ptt_clk_out_ext(uint8_t pin, bool mode)
 			LOG_ERR("Failed to disable (D)PPI channel, error: %08x", err);
 			return false;
 		}
-		nrfx_gpiote_out_uninit(pin);
+		nrfx_gpiote_pin_uninit(pin);
```

Comment:
You need to also free the GPIOTE channel that was allocated for this pin (`task_channel` above).